release notes/v0.38.0.md

```diff
@@ -0,0 +1,180 @@
+
+A few helper functions have been added to the [k6-jslib-utils](https://k6.io/docs/javascript-api/jslib/utils) library for tagging automatically all the metric's samples with the current running stage.
+## Breaking Changes
+
+### `console.log` and `console.info` pretty prints objects and arrays ([2375](https://github.com/grafana/k6/pull/2375))
+
+For convenience, all `console` methods such as `console.log` and `console.info` will now automatically `JSON.stringify` objects and arrays passed to it. Thus, instead of `console.log({'foo': 'bar'})` printing `[object Object]`, it will now print `{'foo': 'bar'}`, which will make the experience of debugging k6 scripts easier and more intuitive. 
+
+The previous behavior can still be achieved by casting the `Object` to a `String`, as in `console.log(String({'foo': 'bar'}))`.  
+
+**Objects**
+
+```javascript
+export default function () {
+  var user = [{}]
+
+  console.log("the user", user) 
+  // before: the user [object Object]
+  // after: the user {"name":"Bob"}
+}
+```
+
+**arrays**
+
+```javascript
+export default function () {
+  var user = {name: "Bob"}
+
+  console.log([1, 2, "mytest", {user: "Bob"}]) 
+  // before: 1,2,mytest,[object Object]
+  // after: [1,2,"mytest",{"user":"Bob"}]
+}
+```
+
+### Deprecations
+
+* [2499](https://github.com/grafana/k6/pull/2499/files) removed support for the deprecated max VU's `max` **option**.
+* This release drops some left overs from the previous version of our JS module APIs. As of **v0.38-0**, these are now unsupported:
+  * The deprecated `common.Bind` ([2488](https://github.com/grafana/k6/pull/2448)) and `common.BindToGlobal` ([2451](https://github.com/grafana/k6/pull/2451)) functions.
+  * The context-based (`common/context.go` [2488](https://github.com/grafana/k6/pull/2448)) utils have also been removed. 
+   
+## Enhancements and UX improvements
+
+### Stricter thresholds' evaluation before the execution starts ([#2330](https://github.com/grafana/k6/issues/2330))
+
+k6 **v0.37.0** already improved thresholds parsing by switching its underlying implementation from JavaScript to Go. k6 **v0.38.0** introduces two additional improvements:
+* k6 will now parse and evaluate thresholds before the execution starts. If a threshold is invalid, as described below, k6 will immediately exit without starting the load test. 
+* k6 will now detect invalid thresholds:
+    ```javascript
+    export const options = {
+      // ...    
+      thresholds: {
+          // Incorrect thresholds expressions:
+          http_req_failed: ['rave<0.01'],  // e.g. "rave" is not a valid aggregation method
+          // Thresholds applying to a non-existing metrics:
+          iDoNotExist: ['p(95)<200'], // e.g. the metric 'iDoNotExist' does not exist
+          // Thresholds applying an aggregation method that's unsupported by the metric they apply to:          
+          my_counter: ['p(95)<200'], // Counter metrics do not support the p() aggregation method
+     },
+    };
```

Comment:
Sorry, I think my suggestion had wrong indentation, it mixed 2 and 4 spaces and next to last bracket has only 1 space. Also, I think GitHub ate the last triple-backticks to end the code block